Investigations/Office Tools.md

```diff
@@ -0,0 +1,204 @@
+
+- Writer is capable of opening and saving documents in a number of formats, including the Open Document (ODT is its default format), MS words Doc,Docx,HTML
+- A spelling and grammar checker
+- Built-in drawing tools
+- Built-in calculation functions
+- Built-in equation editor
+- Export in PDF format, generate hybrid PDF (embedded ODF), create fillable PDF form
+- The ability to import and edit PDF files.
+- Ability to edit HTML, XHTML files visually without using code
+- Export in HTML, XHTML, XML formats
+- Export in EPUB ebook format
+- Contents, index, bibliography
+- Signing documents, encryption
+- Change tracking during revisions, compare two documents
+- Database integration, including a bibliography database
+- MailMerge
+- Scriptable and Remote Controllable via the UNO API
+- Open Type stylistic sets and character variants of fonts are not selectable from the menus, but can be specified manually in the font window. will set the font to stylistic set 6 and choose character variant 3. This is based on the same syntax for [Graphite](https://en.wikipedia.org/wiki/Graphite_(SIL)) font features.
+
+###
+
+
+### Spreadsheets
+
+They are softwares that are used to allow users to store, process and modify data in a tabular form easily. It is an accounting tool to manage tabular data. In a Spreadsheet data is always stored in **cells (**a single unit to input data in spreadsheet softwares).
+
+Their sole purpose is to,
+
+- Simple calculations like addition, average, counting, etc.
+- Preparing charts and graphs on a group of related data
+- Data entry
+- Data formatting
+- Cell formatting
+- Calculations based on logical comparisons
+
+The best examples for spreadsheet softwares are,
+
+- MS-Excel: Retail Cross-Platform spreadsheet package
+- Google Sheets: Freeware Cloud Based spreadsheet package
+- LibreOffice Calc: Open Source spreadsheet package
+
+The most popular spreadsheet software are EtherCalc,libreOffice Calc
+
+- Calc supports 1 million rows in a spreadsheet with macro references to each cell.
+- Calc is capable of opening and saving most spreadsheets in MS excel format. Calc is also capable of saving spreadsheets as PDF files.
+- As with the entire LibreOffice suite, Calc is available for a variety of platforms, including Linux,Mac OS,MS windows and FreeBSD.
+
+Its features includes,
+
+- Ability to read/write Open document (ODF), Excel (XLS), CSV and several other formats.
+- Support for many functions, including those for imaginary numbers, as well as financial and statistical functions.
+- Supports 1 million rows in a spreadsheet, making LibreOffice spreadsheets more suitable for heavier scientific or financial spreadsheets.However, the number of columns is restricted to at most 1024, much lower than Excel&#39;s limit of 16384
+- Up to now, new functions such as IFS, Switch TEXT JOIN, MAXIFS, MINIFS functions, etc. were available only in Excel 2016 and later. LibreOffice Calc can use them.
+
+### Database systems
```

Comment:
Does this comes with office tools?

Reply:
Also libreoffice comes with database solution. I wonder why it is not mentioned.